ledger/accountdb.go

```diff
@@ -1936,7 +1947,12 @@ func (iterator *orderedAccountsIter) Next(ctx context.Context) (acct []accountAd
 	}
 	if iterator.step == oaiStepQueryAccounts {
 		// iterate over the existing accounts
-		iterator.rows, err = iterator.tx.QueryContext(ctx, "SELECT address, data FROM accountbase")
+		iterator.accountBaseRows, err = iterator.tx.QueryContext(ctx, "SELECT rowid, address, data FROM accountbase ORDER BY rowid")
```

Comment:
without an ORDER BY, did it used to implicitly order by rowid? just curious

Reply:
I'm not sure that it would always be the same. especially if use have updates.
there are cases, however, where it will be : if the table is append-only, than the order will remain the same.